examples/meta/src/classifier/shareboost.sg

```diff
@@ -0,0 +1,30 @@
+CSVFile f_feats_train("../../data/classifier_4class_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_4class_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_4class_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_4class_2d_linear_labels_test.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+MulticlassLabels labels_train(f_labels_train)
+MulticlassLabels labels_test(f_labels_test)
+#![create_features]
+
+#![create_instance]
+ShareBoost shareboost(features_train, labels_train, 30)
```

Comment:
The last argument should be `min(fm_train_real.shape[0]-1, 30)`. I dont know where the `30` came from but as the training dataset has 80 rows I just wrote down 30 here.